### PR TITLE
Support Pad-2 in opset-11 ONNX model

### PR DESCRIPTION
**Pull request by mvafin**
Root cause analysis:
Pytorch 1.3 creates an older version of Pad operation even when converting to opset-11.

Solution: 
In order to support such an operation, we have to check if pads exist in attributes.

Ticket: 51378

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR: N/A no transformation changed
* [x]  Transformation preserves original framework node names: N/A no transformation changed

Validation:
* [x]  Unit tests
* [x]  Framework operation tests: N/A no new operation enabled
* [x]  Transformation tests: N/A no transformation changed
* [x]  Model Optimizer IR Reader check

Documentation:
* [x]  Supported frameworks operations list: N/A no new operation enabled
* [x]  Guide on how to convert the **public** model: N/A no new public model enabled
* [x]  User guide update: N/A no new public model enabled

### CODE REVIEW

**Review by pavel-esir — APPROVED** [review]
looks good to me

File: model-optimizer/extensions/front/onnx/pad_ext.py [hunk 1]
```diff
@@ -28,8 +28,10 @@ class PadFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         mode = onnx_attr(node, 'mode', 's', default='constant', dst_type=lambda x: x.decode())
-        if get_onnx_opset_version(node) < 11:
-            pads = onnx_attr(node, 'pads', 'ints', dst_type=lambda x: np.array(x, dtype=np.int64))
+        # Pytorch 1.3 while converting to opset 11, creates Pad from older opset.
+        # To be able to convert such models we have to check if pads attribute exist
```
Comment (rkazants) [inline model-optimizer/extensions/front/onnx/pad_ext.py:32]: exists

File: model-optimizer/extensions/front/onnx/pad_ext.py [hunk 2]
```diff
@@ -28,8 +28,10 @@ class PadFrontExtractor(FrontExtractorOp):
...
+        pads = onnx_attr(node, 'pads', 'ints', dst_type=lambda x: np.array(x, dtype=np.int64))
+        if get_onnx_opset_version(node) < 11 or pads is not None:
             value = onnx_attr(node, 'value', 'f', default=0.)
```
Comment (rkazants) [inline model-optimizer/extensions/front/onnx/pad_ext.py:35]: what is a difference between `pads` and `value`?

File: model-optimizer/extensions/front/onnx/pad_ext.py [hunk 3]
```diff
@@ -28,8 +28,10 @@ class PadFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         mode = onnx_attr(node, 'mode', 's', default='constant', dst_type=lambda x: x.decode())
-        if get_onnx_opset_version(node) < 11:
-            pads = onnx_attr(node, 'pads', 'ints', dst_type=lambda x: np.array(x, dtype=np.int64))
+        # Pytorch 1.3 while converting to opset 11, creates Pad from older opset.
+        # To be able to convert such models we have to check if pads attribute exist
+        pads = onnx_attr(node, 'pads', 'ints', dst_type=lambda x: np.array(x, dtype=np.int64))
```
Comment (rkazants) [inline model-optimizer/extensions/front/onnx/pad_ext.py:33]: dst_type=int64_array

File: model-optimizer/extensions/front/onnx/pad_ext.py (hunk 2 above)
Comment (mvafin) [reply model-optimizer/extensions/front/onnx/pad_ext.py:35]: `pads` specify how to pad the input, `value` specify the value inserted as pads

File: model-optimizer/extensions/front/onnx/pad_ext.py [hunk 4]
```diff
@@ -28,11 +29,13 @@ class PadFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         mode = onnx_attr(node, 'mode', 's', default='constant', dst_type=lambda x: x.decode())
-        if get_onnx_opset_version(node) < 11:
-            pads = onnx_attr(node, 'pads', 'ints', dst_type=lambda x: np.array(x, dtype=np.int64))
+        # Pytorch 1.3 while converting to opset 11, creates Pad from older opset.
+        # To be able to convert such models we have to check if pads attribute exists.
+        pads = onnx_attr(node, 'pads', 'ints', dst_type=int64_array)
+        if get_onnx_opset_version(node) < 11 or pads is not None:
             value = onnx_attr(node, 'value', 'f', default=0.)
```
Comment (rkazants) [inline model-optimizer/extensions/front/onnx/pad_ext.py:36]: what will be if input type is integer then `value` also must be integer according to the specification?

**Review by evolosen — APPROVED** [review]

File: model-optimizer/extensions/front/onnx/pad_ext.py (hunk 4 above)
Comment (mvafin) [reply model-optimizer/extensions/front/onnx/pad_ext.py:36]: `value` is always float for Pad-2 according to [spec](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Pad-2).

File: model-optimizer/extensions/front/onnx/pad_ext.py [hunk 5]
```diff
@@ -28,11 +29,13 @@ class PadFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         mode = onnx_attr(node, 'mode', 's', default='constant', dst_type=lambda x: x.decode())
-        if get_onnx_opset_version(node) < 11:
-            pads = onnx_attr(node, 'pads', 'ints', dst_type=lambda x: np.array(x, dtype=np.int64))
+        # Pytorch 1.3 while converting to opset 11, creates Pad from older opset.
+        # To be able to convert such models we have to check if pads attribute exists.
+        pads = onnx_attr(node, 'pads', 'ints', dst_type=int64_array)
```
Comment (rkazants) [inline model-optimizer/extensions/front/onnx/pad_ext.py:34]: What is about [Pad-1](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Pad-1) that has `paddings` attribute?

Comment (mvafin) [reply model-optimizer/extensions/front/onnx/pad_ext.py:34]: opset-1 is out of our scope, was never supported.

**Review by rkazants — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**